### PR TITLE
Feat/twitter meta tags

### PR DESCRIPTION
**Pull request by edyespinal**
#### What problem is this solving?

This PR adds the messages to a store's settings for the `Twitter Meta Tags` app configuration.

#### How should this be manually tested?

The app is linked in [this workspace](https://twitter--vtexeurope.myvtex.com/admin/cms/store).
You can see the messages on the different available languages for the `Twitter Meta Tags` app configuration.
 

#### Screenshots or example usage
https://user-images.githubusercontent.com/17585823/131830650-c60ba407-149f-4b23-aa36-e45da0146a1b.mov

#### Type of changes

| ✔️  | Type of Change                                                                            |
| --- | ----------------------------------------------------------------------------------------- |
| \_  | Bug fix                                |
| \_  | New feature                        |
| \_  | Breaking change  |
| \_  | Technical improvements  |

### CODE REVIEW

**Comment by edyespinal** [comment]
@vtex-apps/store-framework-devs

**Comment by davicostalf** [comment]
- Instead of including this as an instruction, we should support usernames written both with and without the @.
image

- This Select should have a label, maybe "Twitter card type". Also, please rename the options to follow the name in Twitter's documentation:
image
image

**Comment by edyespinal** [comment]
Hi @davicostalf 👋🏼 
Thanks for the comments!
Will take this into consideration for the PR to the `vtex-apps/store`.
However, this PR is only for adding translations for the messages displayed to the user.

**Comment by davicostalf** [comment]
> Hi @davicostalf 👋🏼 Thanks for the comments! Will take this into consideration for the PR to the `vtex-apps/store`. However, this PR is only for adding translations for the messages displayed to the user.

Thanks. Please send me this other PR in Slack when you have it.
Also, I think these new messages need to be sent to the localization team so they can review and include them in all languages.

**Comment by edyespinal** [comment]
> Thanks. Please send me this other PR in Slack when you have it. Also, I think these new messages need to be sent to the localization team so they can review and include them in all languages.

Hi @davicostalf 
This is the PR to [vtex-apps/store](https://github.com/vtex-apps/store/pull/530).

For this PR the only thing we need is the Localization team translations right?

**Review by vitorflg — APPROVED** [review]

File: react/components/admin/store/StoreForm/index.tsx [hunk 1]
```diff
@@ -43,6 +44,7 @@ const CheckboxWidget = (props: CustomWidgetProps) => (
 const widgets = {
   BaseInput,
   CheckboxWidget,
+  Dropdown,
```
[the remark this answers is not in the document]
Comment (edyespinal) [reply react/components/admin/store/StoreForm/index.tsx]: No, it is not being used @felippenardi 
It was just used to do some tests.
I'll remove it.

**Comment by edyespinal** [comment]
@felippenardi 
Any updates for this PR?

**Pull request merged**